docs/Developers/Debug_Bears.rst

```diff
@@ -122,3 +122,74 @@ Or to debug all bears specified by ``bears`` setting:
     bears = PEP8Bear,MypyBear
     files = <filename>
     debug_bears = True
+
+Inspect Bear settings
+---------------------
+
+External pdb command ``settings`` has been included to inspect the bear
+settings in the debugging environment.(``settings`` command display's the
+updated setting of a Bear)
```

Comment:
The text in the braces shouldn't be in braces, it's essential and not a side note.